helmfile/charts/ingress-nginx-probe-ingress/values.yaml

```diff
@@ -0,0 +1,8 @@
+fullnameOverride: "ingress-nginx-controller-metrics"
```

Comment:
Isn't this fullnameoverride something that is generally not set in helm chart values, where the name is just derived nicely from the release name?
```suggestion
```

Reply:
Yeah, I could change it back and then just hard code this or set it from a separate value
```yaml
paths:
  - backend:
      service:
        name: "ingress-nginx-controller-metrics"
```